js/src/ui/SelectionList/selectionList.css

```diff
@@ -16,22 +16,25 @@
 */
 
 .item {
+  border: 2px solid transparent;
   display: flex;
   flex: 1;
   height: 100%;
   position: relative;
   width: 100%;
 
   &:hover {
-    box-shadow: inset 0 0 0 2px rgb(255, 255, 255);
+    border-color: transparent;
+    filter: none;
+    opacity: 1;
```

Comment:
This just resets everything. Could be specific, i.e. `&.unlselected:hover { ... }` - simpler this way, no impact either way.